modules/capability/module.go

```diff
@@ -71,7 +71,7 @@ func (AppModuleBasic) ValidateGenesis(cdc codec.JSONCodec, config client.TxEncod
 }
 
 // RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the capability module.
-func (a AppModuleBasic) RegisterGRPCGatewayRoutes(_ client.Context, _ *gwruntime.ServeMux) {
+func (a AppModuleBasic) RegisterGRPCGatewayRoutes(ctx client.Context, mux *gwruntime.ServeMux) {
```

Comment:
nit: consistency. unsure if this is the linter just shouting at you randomly tho

Reply:
it isn't.  At some point I figured it may make sense to bring back the _, and started to reinstate them.  Basically because it is instructive to folks what they could / should do.